src/app/qgisapp.cpp

```diff
@@ -10061,11 +10061,7 @@ void QgisApp::duplicateLayers( const QList<QgsMapLayer *> &lyrList )
       QgsVectorLayer *vlayer = qobject_cast<QgsVectorLayer *>( selectedLyr );
       // TODO: add other layer types that can be duplicated
       // currently memory and plugin layers are skipped
```

Comment:
@nyalldawson, after your fix, it shoud be "// currently plugin layers are skipped"